nifi-commons/nifi-processor-utilities/src/main/java/org/apache/nifi/processor/util/put/AbstractPutEventProcessor.java

```diff
@@ -126,8 +127,9 @@
                     + "ensure that the FlowFile content does not contain the delimiter character to avoid errors. If it is not possible to define a delimiter "
                     + "character that is not present in the FlowFile content then the user must use another processor to change the encoding of the data e.g. Base64 "
                     + "encoding.")
-            .required(false)
+            .required(true)
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .defaultValue("\n")
```

Comment:
I think this should be "\\n" so it shows up in the property of the processor, looks like the code already replaces "\\n" to "\n" before using the value